ton_lib/src/tep/meta/metadata/mod.rs

```diff
@@ -0,0 +1,62 @@
+use crate::tep::snake_data::SnakeData;
+use crate::tlb_adapters::DictKeyAdapterTonHash;
+use crate::tlb_adapters::DictValAdapterTLBRef;
+use crate::tlb_adapters::TLBHashMapE;
+use serde::Deserialize;
+use serde::Serialize;
+use serde_json::Value;
+use std::collections::HashMap;
+use std::fmt::Debug;
+use ton_lib_core::cell::{TonCell, TonHash};
+use ton_lib_core::TLBDerive;
+
+mod jetton_metadata;
+mod nft_collection_metadata;
+mod nft_item_metadata;
+
+pub use jetton_metadata::*;
+pub use nft_collection_metadata::*;
+pub use nft_item_metadata::*;
+
+#[derive(PartialEq, Eq, Debug, Clone, TLBDerive)]
+pub enum MetadataContent {
```

Comment:
Personally, I prefer to keep mod.rs wihout any implementations  - so move it to `metadata_content.rs`